x-pack/plugins/infra/public/pages/metrics/hosts/components/search_bar/unified_search_bar.tsx

```diff
@@ -92,27 +86,33 @@ export const UnifiedSearchBar = () => {
           </EuiFlexGroup>
         </EuiFlexItem>
       </EuiFlexGroup>
-      <EuiHorizontalRule margin="none" />
+      <EuiHorizontalRule
+        margin="xs"
+        css={css`
+          margin-bottom: 0;
+        `}
+      />
     </StickyContainer>
   );
 };
 
-const StickyContainer = (props: { children: React.ReactNode }) => {
+const StickyContainer = ({ children }: { children: React.ReactNode }) => {
   const { euiTheme } = useEuiTheme();
   const { headerHeight } = useKibanaHeader();
 
   return (
-    <EuiFlexGrid
-      gutterSize="none"
+    <div
```

Comment:
Nit: here we can use react `Fragment`

Reply:
We need this to be a div in this case, otherwise, the css won't be applied.